examples/armada.yaml

```diff
@@ -20,7 +20,7 @@ armada:
         source:
           type: git
           location: git://github.com/att-comdev/openstack-helm
-          subpath: common
+          subpath: helm-toolkit
```

Comment:
This is a much needed change to reflect the rename.  One small nit, I would recommend changing all "common" references along with this because the goal with that rename in the `openstack-helm` repository was to avoid name collisions and as it stands, this will result in a release named `common` and it would be better to get everything using `helm-toolkit`